symphony-bdk-legacy/symphony-api-client-java/src/test/java/it/clients/symphony/api/HealthcheckClientTest.java

```diff
@@ -35,10 +34,24 @@ public void performHealthCheckSuccess() {
         "\"ceServiceUser\": true\r\n" +
         "}");
 
-    HealthcheckResponse response = healthCheckClient.performHealthCheck();
+    try {
 
-    assertNotNull(response);
-    assertEquals(true, response.getPodConnectivity());
+      assertNotNull(healthCheckClient);
+
+      final HealthcheckResponse response = healthCheckClient.performHealthCheck();
+      assertNotNull(response);
+
+      assertTrue(response.getPodConnectivity());
+      assertTrue(response.getKeyManagerConnectivity());
+      assertTrue(response.getEncryptDecryptSuccess());
+      assertEquals("1.54.1", response.getPodVersion());
+      assertEquals("2.54.0", response.getAgentVersion());
+      assertTrue(response.getAgentServiceUser());
+      assertTrue(response.getCeServiceUser());
+
+    } catch (Exception e) {
```

Comment:
An exception raised during a test will fail it, so we don't need to try/catch them

Reply:
Ok thanks, I will fix it.

Otherwise in the method **performHealthCheckSuccessWithUnknownField()** I don't know how to get the value of the field **unknownField**. I didn't find a getter for this field in the response object.